lib/main.dart

```diff
@@ -13,113 +16,294 @@ class MyApp extends StatelessWidget {
+              margin: const EdgeInsets.only(top: 40),
+              padding: const EdgeInsets.only(left: 50),
+              child: const Text('Zaloguj sie',
+                  style: TextStyle(
+                    fontSize: 20,
+                    fontWeight: FontWeight.w600,
+                  ))),
+          Align(
+            alignment: FractionalOffset.topLeft,
+            child: Container(
+              width: 120,
+              margin: const EdgeInsets.only(left: 50),
+              height: 2,
+              decoration: BoxDecoration(
+                color: AppColors.primaryColor,
+                borderRadius: BorderRadius.circular(10),
+              ),
+            ),
+          ),
+          Container(
+            margin: const EdgeInsets.only(top: 35, left: 50, right: 50),
+            child: Form(
+              key: _formKey,
+              child: Column(children: [
+                TextFormField(
+                  controller: _textController,
+                  decoration: InputDecoration(
+                      labelText: "Email",
+                      labelStyle: const TextStyle(
+                          color: AppColors.grayAccent, fontSize: 15),
+                      focusedBorder: OutlineInputBorder(
+                          borderRadius: BorderRadius.circular(12.0),
+                          borderSide:
+                              const BorderSide(color: AppColors.primaryColor)),
+                      enabledBorder: OutlineInputBorder(
+                        borderRadius: BorderRadius.circular(12.0),
+                        borderSide: const BorderSide(
+                          color: AppColors.grayAccent,
+                        ),
+                      ),
+                      contentPadding: const EdgeInsets.all(15)),
+                ),
+                const SizedBox(height: 20.0),
+                TextFormField(
+                  controller: _passController,
+                  decoration: InputDecoration(
+                      labelText: "Hasło",
+                      labelStyle: const TextStyle(
+                          color: AppColors.grayAccent, fontSize: 15),
+                      focusedBorder: OutlineInputBorder(
+                          borderRadius: BorderRadius.circular(12.0),
+                          borderSide:
+                              const BorderSide(color: AppColors.primaryColor)),
+                      enabledBorder: OutlineInputBorder(
+                        borderRadius: BorderRadius.circular(12.0),
+                        borderSide: const BorderSide(
+                          color: AppColors.grayAccent,
+                        ),
+                      ),
+                      contentPadding: const EdgeInsets.all(15)),
+                  obscureText: true,
+                ),
+                Container(
+                  width: 400,
+                  margin: const EdgeInsets.only(top: 20),
+                  child: ElevatedButton(
```

Comment:
This could be extracted as 'LoginButton'
also this is very similar to 'RegisterButton', a common base could be probably extracted